packages/nx/src/native/tui/components/help_popup.rs

```diff
@@ -9,6 +9,9 @@ use ratatui::{
     },
 };
 use std::any::Any;
+use tokio::sync::mpsc::UnboundedSender;
```

Comment:
Do the crossbeam senders work? Why use the tokio ones? We mostly use the crossbeam senders elsewhere in the codebase.